Automattic/wp-calypso
# Update - add option to filter reader by unseen posts

#### Changes proposed in this Pull Request

This PR takes a stab at adding a toggle-able control on Reader list pages to allow the user to hide all posts which they have already seen.

#### Testing instructions



* Navigate to `/read`
* Verify that an "Unread" menu option shows in the sidebar under "Following" and "A8C".
* Navigate to these views and verify that already-read posts appear and disappear as expected.

![image](https://user-images.githubusercontent.com/13437011/113445555-b59c0300-93bb-11eb-8c80-7597ddaa23af.png)
![image](https://user-images.githubusercontent.com/13437011/113445582-bf256b00-93bb-11eb-9aa4-7e1eeb8c5ad9.png)



Related to (no corresponding issue - 20% time task)
